salt/modules/disk.py

```diff
@@ -464,11 +479,18 @@ def fstype(device):
     if salt.utils.which('df'):
         # the fstype was not set on the block device, so inspect the filesystem
         # itself for its type
-        df_out = __salt__['cmd.run']('df -T {0}'.format(device)).splitlines()
-        if len(df_out) > 1:
-            fs_type = df_out[1]
-            if fs_type:
-                return fs_type
+        if __grains__['kernel'] == 'AIX' and os.stat('/usr/sysv/bin/df'):
```

Comment:
The file should exist always, but I thought I'd check just in case, and generate the OSError for the case were someone has really messed with their AIX installation.  What would you recommend instead.